.github/workflows/molecule-install-xnat.yml

```diff
@@ -6,6 +6,7 @@ on:
       - "roles/xnat_container_service/**"
       - "playbooks/install_xnat.yml"
       - "playbooks/install_container_service.yml"
+      - "playbooks/molecule/**/xnat/**"
```

Comment:
ensure the tests run whenever the molecule molecule configuration or inventory changes